scipy/optimize/_bglu_dense_py.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python3
```

Comment:
Of course I created this first and then modified it to get the Cython version. I kept this around because it was useful for profiling and debugging. We're going to want to add functionality to the BGLU class in the future, and it would be useful to have this file around still, I think. Should I just keep it myself? Is this not a common thing to want?